ex12/main.go

```diff
@@ -2,12 +2,10 @@ package main
 
 import "fmt"
 
-type void struct{}
-
 func main() {
 	a := []string{"aboba", "abeba", "aboba", "abeba", "abeba"}
-	set := make(map[string]void)
-	var member void
+	set := make(map[string]struct{})
+	var member struct{}
 
 	for _, v := range a {
 		set[v] = member
```

Comment:
If you want to create an empty struct you don't need to define a variable for that
```suggestion
	for _, v := range a {
		set[v] = struct{}
```